src/path.rs

```diff
@@ -136,38 +142,34 @@ impl Path {
     // Path funcs
 
     /// Starts new sub-path with specified point as first point.
-    pub fn move_to(&mut self, x: f32, y: f32) {
-        self.append(&[PackedVerb::MoveTo], &[x, y]);
+    pub fn move_to<T: Into<[f32; 2]>>(&mut self, pos: T) {
```

Comment:
I'd prefer: `pub fn move_to(&mut self, pos: impl Into<[f32; 2]>) {`

Reply:
I also got this idea after I pushed.